core/src/main/java/com/graphhopper/routing/util/spatialrules/countries/DefaultSpatialRule.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.routing.util.spatialrules.countries;
+
+import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.routing.util.spatialrules.AbstractSpatialRule;
+import com.graphhopper.routing.util.spatialrules.AccessValue;
+
+/**
+ * Default Implementation for the SpatialRule that contains the current default Values
+ */
+public class DefaultSpatialRule extends AbstractSpatialRule {
```

Comment:
Shouldn't this be part of the AbstractSpatialRule, e.g. getCountryIsoA3Name implementation is still missing and so this is not really a default (?)